Freshli.Web/Controllers/AnalysisRequestsController.cs

```diff
@@ -5,21 +5,28 @@
 using Freshli.Web.Models;
 using Hangfire;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.EntityFrameworkCore;
 using XPlot.Plotly;
 
 namespace Freshli.Web.Controllers {
-  [Route("[controller]")]
```

Comment:
Including this annotation was preventing the `Create` view from loading.

Reply:
I think that's because a `HttpGet` attribute was missing. I added one in 89e74ed.